### DISCUSSION TITLE
none of the sprites i have downloaded work even though i have tried all the instructions please help

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#9133] [general - spritepacks]
none of the sprites i have downloaded work even though i have tried all the instructions please help

**Comment by Wingdinggaster656** [comment]
Please describe your problem in detail.
What do you mean by "the instructions"? And what do you mean by "none of the sprites I have downloaded work"?

Do you know that some sprites just enable a gift function, without gifting, they are meaningless?

**Reply by ghost** [reply]
the sprites i downloaded are from the sprite pack page ,and I have gifted them to her but after her wearing it for a bit it doesn't let me choose them for her again, i have tried gifted them to her again but it doesnt work either
the only sprites that work are her ribbons

**Reply by Wingdinggaster656** [reply]
Your affection range?
Monika will not just do what you say if your affection isn't that high. So your problem doesn't sound like a real problem, more like a normal effect.

**Reply by ghost** [reply]
ohh i see, could you tell me how to check affection? thank you for your help tho

**Reply by Wingdinggaster656** [reply]
There is a `/log` folder, not sure if you have noticed it.

Inside that folder, there is a `aff_log.txt`, which means affection log.
You could find how your affection was changed all the time there.

**Reply by ghost** [reply]
how much affection do you need to unlock stuff like that

**Reply by Wingdinggaster656** [reply]
Not really sure, but I know when your affection is above 1000, you can ask her to wear whatever she wore.

**Reply by Booplicate** [reply]
You need 30 affection.
